server/utils/jwt/index.ts

```diff
@@ -0,0 +1,26 @@
+import jwt from 'jsonwebtoken';
+
+const verifyToken = (token: string, secret: string) => new Promise((resolve, reject) => {
+  jwt.verify(token, secret, (err, decode) => {
+    if (err) {
+      reject(err);
+    } else {
+      resolve(decode);
+    }
+  });
+});
+
+const signToken = (payload:string, secret:string) => new Promise((resolve, reject) => {
```

Comment:
Do you need to send the secret as an argument? your secret is in the .env which you can reach from here.